ci/keystone/keystone.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+#
+# Helper script for setting up and running keystone. Most of this is taken from:
+#
+#   http://docs.openstack.org/developer/keystone/developing.html
+
+# tunable variables:
+
+# So we can avoid repeatedly downloading the repo when developing this script:
+keystone_repo=${keystone_repo:-https://git.openstack.org/openstack/keystone}
+
+# git commit to use. Travis uses this to test against different releases of
+# keystone:
+keystone_commit=${keystone_commit:-master}
```

Comment:
I'm not as concerned about the API being stable as I am about the checked in keystone.conf :-)
My concern is that a random future PR could experience test failures for something outside the submitter's control.

@zenhack - could we use `stable/mitaka` here?